### PR TITLE
rework layout

### PR DESCRIPTION
**Pull request by notmd**
Our layout now uses `position: fixed` instead of tricky `display: grid` and `over-flow: auto` to make content scroll. Manually verified all pages that there is no regression. The UI still looks the same as before.

### CODE REVIEW

File: website/src/components/SideMenu.tsx [hunk 1]
```diff
@@ -1,75 +1,84 @@
...
+              className="hidden lg:hidden sm:block"
+            >
+              <Link
                 key={item.labelID}
-                fontFamily="inter"
-                label={label}
-                placement="right"
-                className="hidden lg:hidden sm:block"
+                href={item.pathname}
+                target={!item.target ? "_self" : item.target}
```
Comment (AbdBarho) [inline website/src/components/SideMenu.tsx:57]: `item.target ?? "_self"`

**Review by AbdBarho — COMMENTED** [review]
Thank you, looks good on mobile?

**Comment by notmd** [comment]
> Thank you, looks good on mobile?

Yes. On mobile the navbar is now fixed too

File: website/src/components/SideMenu.tsx (hunk 1 above)
Comment (notmd) [reply website/src/components/SideMenu.tsx:57]: > `item.target ?? "_self"`

That's is the old code, I didn't want to touch it.

**Review by AbdBarho — APPROVED** [review]

**Pull request merged**